<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBetrieb</class>
 <widget class="QWidget" name="FormBetrieb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>613</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>170</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>170</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>165</red>
           <green>167</green>
           <blue>169</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Betriebe</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="nameEdit">
        <property name="placeholderText">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>30</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>30</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>165</red>
              <green>167</green>
              <blue>169</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>Nr:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QTableWidget" name="lehrlingTableWidget"/>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLineEdit" name="phoneEdit">
        <property name="placeholderText">
         <string>Telefon</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLineEdit" name="ortEdit">
        <property name="placeholderText">
         <string>PLZ / Ort</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nrBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLineEdit" name="strasseEdit">
        <property name="placeholderText">
         <string>Straße</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QPushButton" name="deleteLehrlingButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Lehrling vom Betrieb löschen</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>30</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>30</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>165</red>
              <green>167</green>
              <blue>169</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>Auszubildende:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>30</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>30</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>165</red>
              <green>167</green>
              <blue>169</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>Sortieren:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="sortBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="betriebeTableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>30</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>30</red>
              <green>170</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>165</red>
              <green>167</green>
              <blue>169</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>Letzte Änderung:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="lastChangeEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Löschen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changeButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Ändern</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="text">
         <string>&amp;Neu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>170</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>170</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>165</red>
              <green>167</green>
              <blue>169</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>S&amp;peichern</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>&amp;Schließen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>strasseEdit</tabstop>
  <tabstop>ortEdit</tabstop>
  <tabstop>phoneEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>betriebeTableWidget</tabstop>
  <tabstop>lehrlingTableWidget</tabstop>
  <tabstop>deleteLehrlingButton</tabstop>
  <tabstop>nrBox</tabstop>
  <tabstop>sortBox</tabstop>
  <tabstop>changeButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>lastChangeEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
